config/routes.rb

```diff
@@ -32,11 +32,12 @@
       member do
         put :start_calibration
         put :stop_calibration
+
+        resources :sensor_readings, only:[:show, :create, :index], default: { format: :json }
+        post 'sensor_readings/debug', to: 'sensor_readings#debug', default: { format: :json }
```

Comment:
@kabel42 I moved the `sensor_readings` route up to `/reports/:report_id/sensors/:id/sensor_readings`. If you don't know the correct `:id` of our sensor, just use any number and override it via `address` or `name` in the supplied params.